engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-details.html

```diff
@@ -123,6 +123,26 @@
           </li>
         </ul>
       </li>
+      <li uib-dropdown
+          ng-class="{active: stateIncludes('content-host.debs-actions') || stateIncludes('content-host.debs-installed') || stateIncludes('content-host.debs-upgradeable')}" >
+        <a uib-dropdown-toggle>
+          <span translate>Deb Packages</span>
```

Comment:
This UI portion is where I'm going to ask for the most changes. Apologies in advance because I know I had asked you to go and create some tests which may no longer be needed.

What we would like to see is the Debian package list & actions be implemented within the existing views under the Packages tab. Due to the different data for an ::InstalledPackage and ::InstalledDeb there will need to be some logic along the lines of:

for debian:
  render the debian installed packages table (communicate with your new API)
else:
  do what is there today with existing views and API

@johnpmitsch may be able to give you some pointers on how to do this the "angular way"!

Reply:
Let's not go this route.